server/controllers/availability.js

```diff
@@ -0,0 +1,139 @@
+const PetSitter = require("../models/PetSitter")
+const Schedule = require("../models/Schedule");
+const asyncHandler = require("express-async-handler");
+const Availability = require("../models/Availability");
+
+// @route POST /availability/schedule/:scheduleId
+// @desc create or update a schedule
+// @access Private
+exports.createSchedule = asyncHandler(async (req, res, next) => {
+  const petsitter = await PetSitter.findOne(req.user);
+  const availability = await Availability.findById(petsitter.availabilityId);
+  const scheduleIds = await availability.scheduleIds
+  const { name, MondayFrom, MondayTo, TuesdayFrom, TuesdayTo, WednesdayFrom, WednesdayTo, ThursdayFrom, ThursdayTo, FridayFrom, FridayTo, SaturdayFrom, SaturdayTo, SundayFrom, SundayTo } = req.body;
```

Comment:
Not a huge fan of the PascalCase on this here, any chance we can do camelCase instead 🙂

Reply:
updated - thanks!!